docs/enterprise-guide/serialization.md

```diff
@@ -0,0 +1,60 @@
+## Copying contents of one Metabase instance to another
+
+Metabase's serialization feature allows you to create a snapshot, called a dump, of the contents of a Metabase instance that can then be loaded into another instance.
+
+This lets you do things like create a set of dashboards in one Metabase instance and then easily copy those dashboards to a number of other Metabase instances that you've set up for your customers. You could also use this feature to enable a staging-to-production workflow for important dashboards or reports by dumping from a staging instance of Metabase and then loading that dump into your production instance(s).
```

Comment:
Might be good to clarify whether this is also meant for backups or should backups be done by backing up the app DB.

Also, should we mention that the dumps are human-readable (ish) YAMLs which you can put under source control and audit?